packages/Python/lldbsuite/test/lang/swift/resilience/TestResilience.py

```diff
@@ -106,6 +106,11 @@ def doTestWithFlavor(self, exe_flavor, mod_flavor):
 
         source_name = "main.swift"
         source_spec = lldb.SBFileSpec(source_name)
+
+        target = self.dbg.CreateTarget(exe_path)
```

Comment:
I assume the creating the target twice is necessary here ro at least doesn't break anything?

Reply:
It didn't break anything in my tests, and I kinda expected the run_to* helpers to pick up an existing target if there was one, but it doesn't look like that do. Now I wonder how this works...

Reply:
Ok, registerSharedLibrariesWithTarget() adds the remote file location to the Module which is persistent across targets. So this doesn't hurt.

Reply:
I think I have an idea how to improve this. I'll try it tomorrow.